pyproject.toml

```diff
@@ -21,7 +21,7 @@ charm-refresh = "^3.1.0.2"
 httpx = "^0.28.1"
 charmlibs-snap = "^1.0.1"
 charmlibs-interfaces-tls-certificates = "^1.1.0"
-postgresql-charms-single-kernel = "16.1.4"
+postgresql-charms-single-kernel = {url = "https://github.com/canonical/postgresql-single-kernel-library/archive/7014727ccc05a0463a301ffbcf8c270d419f9e8b.zip"}
```

Comment:
This will be updated to "16.1.5" when https://github.com/canonical/postgresql-single-kernel-library/pull/54 is merged.